src/EventGridExtension/OutputBinding/EventGridAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Microsoft.Azure.WebJobs.Description;
+
+namespace Microsoft.Azure.WebJobs.Extensions.EventGrid
+{
+    /// <summary></summary>
+    /// <seealso cref="System.Attribute" />
+    [AttributeUsage(AttributeTargets.Parameter | AttributeTargets.ReturnValue)]
+    [Binding]
+    public sealed class EventGridAttribute : Attribute
+    {
+        /// <summary>Gets or sets the topic events endpoint URI. Eg: https://topic1.westus2-1.eventgrid.azure.net/api/events </summary>
+        [AppSetting]
+        public string TopicEndpointUri { get; set; }
+
+        /// <summary>Gets or sets the sas key setting.</summary>
+        [AppSetting]
+        public string SasKey { get; set; }
```

Comment:
Rename this to SasKeySetting to make it clear that this isn't supposed to be set by user to the actual key value, but the name of an app setting. All our other bindings follow this pattern.

Reply:
Also, please fix up/improve the comments here, since they will be used by users via intellisense. E.g. for reference https://github.com/Azure/azure-webjobs-sdk-extensions/blob/cdeb65faacf04c53845fb76c4b513c9998fc97de/src/WebJobs.Extensions.Twilio/TwilioSMSAttribute.cs#L22